plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/wizard/beginner/BeginnerTaskQuestionPage.java

```diff
@@ -219,6 +220,7 @@ private void createQuestionControl(final Composite parent, final Question questi
 				new Label(parent, SWT.NONE);
 				//added description for questions
 				if (!question.getNote().isEmpty()) {
+					

```

Comment:
Remove. Also please review the files before committing and only commit meaningful modifications.